### ISSUE TITLE
Use the actual path to plugin zip in scripts/verify-artifact-contents

### ISSUE THREAD

**Issue by PawelLipski** [VirtusLab/git-machete-intellij-plugin#1154]
As for now, `plugin_zip` is inferred as

```shell
plugin_zip=$(ls -1t build/distributions/git-machete-intellij-plugin-${plugin_version}*.zip | head -1)
```

Let's instead add a Gradle task that would print out the zip path (see how `IDEPROBE_INTELLIJ_PLUGIN_URI` is set up in UiTests.kt and use that path here, instead of relying on the latest file from build/distributions/.

**Comment by LukaszKontowski** [comment]
> Let's instead add a Gradle task that would print out the zip path (see how IDEPROBE_INTELLIJ_PLUGIN_URI is set up in UiTests.kt) and use that path here, instead of relying on the latest file from build/distributions/.

Indeed - the `IDEPROBE_INTELLIJ_PLUGIN_URI` is the value that we could use directly instead of the result of
`$(ls -1t build/distributions/git-machete-intellij-plugin-${plugin_version}*.zip | head -1)`

And it looks like it should be available when `./scripts/verify-artifact-contents` is invoked - since it is invoked after first invocation of `./gradlew` (which invokes the `configureUiTests()` function from `UiTests.kt`).

But for some reason, the `IDEPROBE_INTELLIJ_PLUGIN_URI` environment variable is not available when `./scripts/verify-artifact-contents` is invoked.

**Comment by LukaszKontowski** [comment]
Probably `IDEPROBE_INTELLIJ_PLUGIN_URI` will be available only for processes invoked by `./gradlew` (but I am not 100% sure)

**Comment by PawelLipski** [comment]
> Probably IDEPROBE_INTELLIJ_PLUGIN_URI will be available only for processes invoked by ./gradlew (but I am not 100% sure)

Yes exactly, no process is able to export an environment variable **into the other processes** running in the same shell (i.e. to **sibling** processes, or to b process). Env vars can only be passed to **child** processes. When Gradle forks a process (like a JVM with UI tests), then it can fill up its environment with vars like `IDEPROBE_INTELLIJ_PLUGIN_URI`. There is no (non-hackish) way, however, for these vars to be made available to the processes that follow Gradle in the same shell, like `scripts/verify-artifact-contents`.

**Comment by PawelLipski** [comment]
In fact it'd be quite a massive security vulnerability (and also a recipe for chaos) if processes could modify the environment of the calling process (like a shell) 😱
